FND/Agreements/Contracts.rdf

```diff
@@ -325,43 +333,40 @@ A unilateral contract differs from a Bilateral Contract, in which the parties ex
 		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-dt-fd;hasDate"/>
 		<rdfs:label>has effective date</rdfs:label>
 		<rdfs:range rdf:resource="&fibo-fnd-dt-fd;Date"/>
-		<skos:definition>the date a contract, relationship, or policy comes into force</skos:definition>
+		<skos:definition>indicates the date a contract, relationship, or policy comes into force</skos:definition>
 	</owl:ObjectProperty>
 	
 	<owl:ObjectProperty rdf:about="&fibo-fnd-agr-ctr;hasExecutionDate">
 		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-dt-fd;hasDate"/>
 		<rdfs:label>has execution date</rdfs:label>
 		<rdfs:range rdf:resource="&fibo-fnd-dt-fd;Date"/>
-		<skos:definition>the date on which a contract has been fully performed by all parties</skos:definition>
-		<fibo-fnd-utl-av:adaptedFrom rdf:resource="https://definitions.uslegal.com/e/executed-contract/"/>
-		<fibo-fnd-utl-av:explanatoryNote>In other words, the execution date is the date on which the terms of the contract have been completely fulfilled.</fibo-fnd-utl-av:explanatoryNote>
+		<skos:definition>indicates the date on which a contract has been signed by all the necessary parties</skos:definition>
```

Comment:
Does this mean the final party if they do not all sign on the same day?
It seems that timezones could come into play - signing on a certain date in one TZ could be a different date in another. hasExecutionDateTimeStamp seems far more precise - so why is this hasExecutionDate property even needed?

Reply:
It's important for swaps, among other things.  The date that you sign / execute a contract is often different from the date that it comes into force.  Many systems only record the date, so this is optional to provide various formats.

Reply:
The standard interpretation is that the execution date is the date that the "last" party has signed off, meaning, on which the contract is fully executed by all parties.

Reply:
My question still applies - why do we need 2 properties hasExecutionDate and the (more precisely defined) hasExecutionDateTimestamp? Neither refer to the date the contract comes into force, which I agree is different. At the very least the definition for hasExecutionDate should make clear which TZ applies (e.g. that of the signatory or that of the originator). This could make a significant legal difference if it means the contract is deemed to have been executed in a different month/quarter/year.

Reply:
For most contracts, hasExecutionDate is a date and the timestamp is immaterial.  There are cases, such as for securities transactions, where the time stamp is important however.  If we are to provide support to most FIBO users, then we need both.  The date time stamp format allows for time zone - look at the XML Schema Datatypes to see how it works fo rthe latter case.  I have added another property, hasEffectiveDateTimeStamp, to account for the case we are missing but am reticent to remove anything.  They really are use case specific and we have use cases for both in FIBO, for the effective date as well as for the execution date.

Reply:
See the latest version for my approach to supporting either format for both effective date and execution date.